OpenRA.Mods.Common/Scripting/Global/WidgetGlobal.cs

```diff
@@ -0,0 +1,35 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Drawing;
+using OpenRA.Graphics;
+using OpenRA.Scripting;
+using OpenRA.Mods.Common.Widgets;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Scripting.Global
+{
+	[ScriptGlobal("MissionText")]
```

Comment:
How about `Widget`? That is in line with `WidgetGlobal`, removes redundancy as it becomes `Widget.SetMissionText` instead of `MissionText.SetMissionText` and avoids abbreviations which always look weird in CamelCase.